eclipse-cdt-cloud/vscode-memory-inspector
# Add configuration for display options

#### What it does


Addresses second bullet point in #27  by adding configuration options to persist column display and byte display preferences.
- Fixes #36   
- Fixes #30 


#### How to test
- Get a running debug session going and open the memory inspector
- open the preferences view and search "memory inspector"
- toggle the new preferences for variables/ascii display as well as wordsPerGroup/groupsPerRow, observe that behave as expected
- Relaunch the extension, observe that the extension starts up with your settings


#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
